pages/empresas/atualizar.vue

```diff
@@ -94,6 +94,14 @@ export default {
     },
   },
 
+  watch: {
+    errors() {
+      if (this.errors.length > 0) {
```

Comment:
Um watcher recebe dois argumentos representando o novo valor e o valor antigo (não lembro a ordem). Eu acho que é melhor adotá-los, ao invés de checar com `this.errors`, porque pode dar alguma condição de corrida.